res/Home.tsx

```diff
@@ -0,0 +1,14 @@
+import Svg, {Path} from 'react-native-svg';
+
+const HomeIcon = ({color}) => (
```

Comment:
Estos SVG los podes definir como .svg nomas, lo que estar retornando en el componente digamos

`<Svg>
  .....
</Svg>
`

Despues los podes importar como si fueran un componente 
El mismo comentario para los componentes que tenes abajo, que tambien son svg

Reply:
Va un ejemplo de como cambiarle el color siguiendo esto que te comenté arriba
https://stackoverflow.com/questions/49660912/react-native-how-to-use-local-svg-file-and-color-it

Reply:
si, probé con el transformer de react antive svg pero no pude. Entiendo que tiene sentido que no se pueda, pues si le cambias el color a un svg, a que path se lo aplica?